src/proto/vim9class.pro

```diff
@@ -31,6 +31,9 @@ void object_free_items(int copyID);
 void emsg_var_cl_define(char *msg, char_u *name, size_t len, class_T *cl);
 void method_not_found_msg(class_T *cl, vartype_T v_type, char_u *name, size_t len);
 void member_not_found_msg(class_T *cl, vartype_T v_type, char_u *name, size_t len);
+void defcompile_class(class_T *cl);
+void defcompile_classes_in_script();
```

Comment:
```suggestion
void defcompile_classes_in_script(void);
```

Reply:
Thanks.  Incorporated in the latest update to the PR.